packages/components/CallToActionSection.tsx

```diff
@@ -19,12 +20,12 @@ export default function CallToActionSection() {
           <Trans
             i18nKey={t("title")}
             components={{
-              span: <span className="text-purple-400" />,
+              span: <span className={classes("text-purple-400")} />,
```

Comment:
Not needed here, We only use `classes` when we wanna (conditionally) join classNames

```suggestion
              span: <span className="text-purple-400" />,
```